viewer/core/src/datamodels/cad/CadModelFactory.test.ts

```diff
@@ -4,11 +4,10 @@
 import * as THREE from 'three';
 
 import { Mutable } from '../../utilities/reflection';
-import { createCadModelMetadata } from '../../__testutilities__/createCadModelMetadata';
-import { generateSectorTree } from '../../__testutilities__/createSectorMetadata';
+import { generateSectorTree, createCadModelMetadata } from '../../../../test-utilities';
```

Comment:
Should test-utilities be a package? Not sure how that will work dependency-wise though.

Reply:
I discussed a bit with Christopher, I ended up with not having it as a package. Instead it is just a folder without a package.json that is referenced from all the tests that uses it. Open to suggestions for how to do this better

Reply:
Ok, lets leave it as is